src/Language/PureScript/Parser/Declarations.hs

```diff
@@ -380,7 +380,7 @@ parseLet = do
   indented
   ds <- mark $ P.many1 (same *> parseLocalDeclaration)
   indented
-  reserved "in"
+  reserved "in" P.<?> "\"in\" (for declaration in PSCi, try without \"let\")"
```

Comment:
I was hoping this could go in the PSCi parser actually. It seems like a shame to have to include it in the regular compiler and elsewhere.